docs/content/concepts/assets/asset-selection-syntax.mdx

```diff
@@ -1,32 +1,549 @@
 ---
-title: Asset Selection Syntax | Dagster
-description: To specify an asset selection, Dagster supports a simple query syntax.
+title: "Asset selection syntax | Dagster Docs"
+description: "Learn how to use Dagster's query syntax to select assets in your code, the Dagster UI, and the Dagster CLI."
 ---
 
-# Asset Selection Syntax
+# Asset selection syntax
 
-To specify an asset selection as a string, Dagster supports a simple query syntax. This selection syntax is accepted in a few different places:
+Using a simple query syntax, you can specify an asset selection as a string. In this guide, we'll cover where the syntax is supported, how it works, and some examples.
 
-- The `list` and `materialize` commands in the [asset command-line interface](/\_apidocs/cli#dagster-asset).
-- The asset filter text box on the asset graph page, in the UI.
-- The `selection` parameter of <PyObject object="define_asset_job" />. (This parameter alternatively accepts an <PyObject object="AssetSelection"/> object, which supports more complex selections built from compositions of Python objects.)
+---
+
+## Supported locations
+
+The asset selection syntax can be used in:
+
+- **In the `selection` parameter of <PyObject object="define_asset_job" />.** Alternatively, this parameter accepts an <PyObject object="AssetSelection"/> object, which supports more complex selections built from compositions of Python objects. For example:
+
+  ```python
+  taxi_zones_job = define_asset_job(name="taxi_zones_job", selection="taxi_zones_file")
+  ```
+
+- **The [asset command-line interface](/\_apidocs/cli#dagster-asset)** with the `list` and `materialize` commands. For example:
+
+  ```shell
+  dagster asset list --select taxi_zones_file
+  ```
+
+- **In the Dagster UI**, in the search box on the [**Global asset lineage** page](/concepts/webserver/ui#global-asset-lineage). The [Examples](#examples) section demonstrates how asset selection queries work in the UI.
+
+---
+
+## Usage
+
+A query includes a list of clauses. Clauses are separated by commas, except in the case of the `selection` parameter of <PyObject object="define_asset_job" />, <PyObject object="materialize" />, and <PyObject object="materialize_to_memory" />, where each clause is a separate element in a list.
+
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
+    <tr>
+      <th
+        style={{
+          width: "15%",
+        }}
+      >
+        Clause syntax
+      </th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <code>ASSET_KEY</code>
+      </td>
+      <td>Selects a single asset by asset key</td>
+    </tr>
+    <tr>
+      <td>
+        <code>COMPONENT/COMPONENT</code>
```

Comment:
Ideas for a better way to convey this? Does it always involve a prefix (and a suffix?)

Reply:
This made sense to me when i read it. I think no need to mention "suffix" since we dont really use that descriptor anywhere else (that i'm aware of). But prefix is mentioned all the time so i think that makes sense to include here